nautobot/core/tests/test_authentication.py

```diff
@@ -383,7 +383,16 @@ def test_create_object(self):
         )
         obj_perm.users.add(self.user)
         obj_perm.object_types.add(ContentType.objects.get_for_model(Prefix))
-
+        related_obj_perm = ObjectPermission.objects.create(
+            name="Related object permission",
+            actions=["view", "add", "change"],
```

Comment:
Are all three actions actually needed here (and if so, why?) or can we narrow this in scope? Same question below.

Reply:
Seems like "view" action is sufficient here. I will remove the rest.